app/views/content/blog/marias-return-to-teaching-story.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Maria's return to teaching story
+author: Maria Bateson
+date: "2021-04-21"
+images:
+    maria:
+    path: "media/images/content/blog/maria.jpg"
+    alt: "A photograph of Maria's face"
```

Comment:
I'm afraid I started a bad trend by prefixing alt text with 'A photograph of' :(

I believe this should be something more along the lines of

```suggestion
    alt: "Returning teacher, Maria Bateson"
```